examples/window_debug.rs

```diff
@@ -139,6 +142,21 @@ fn main() {
           content_protection = !content_protection;
           window.set_content_protection(content_protection);
         }
+        "M" => {
+          let minimizable = !window.is_minimizable();
+          println!("Minimizable: {}", minimizable);
```

Comment:
remove

Reply:
I'm confused, I thought you asked me to move the examples of usage of these new APIs to here. Do you want me to remove all the examples altogether, or do you want them somewhere else?

Reply:
no just remove this println! call

Reply:
Ah, got it! Ok, will do.